mcp-server/docs/setup/QUICK-START-DOCKER-PAT.md

```
@@ -0,0 +1,120 @@
+# 🚀 Quick Start: Cursor + Docker + PAT
+
+Quick guide for setting up Cursor to connect to MCP server in Docker container using Personal Access Token.
+
+## Your Token
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)